ebms-send-in/src/main/kotlin/no/nav/emottak/ebms/App.kt

```diff
@@ -28,20 +28,21 @@ fun main() = SuspendApp {
             awaitCancellation()
         }
     }.onFailure { error ->
-        if (error !is CancellationException) logError(error)
+        if (error !is CancellationException) {
+            log.error("Shutdown ebms-send-in due to: ${error.stackTraceToString()}")
+        }
```

Comment:
For å være ærlig er jeg litt agnostisk til om loggingen gjøres i en egen metode - hvis noen har sterke formeninger om denne bør være i en egen metode kan vi gjøre det. Tenkte i utgangspunktet at det var unødvendig.